ablation/distributions.py

```diff
@@ -8,10 +8,12 @@
 from numpy.random import permutation, randn
 from scipy import stats
 from scipy.ndimage import gaussian_filter
-from shap import sample
+
+# from shap import sample
```

Comment:
Reason to keep this commented and not just remove it?

Reply:
yes forgot thx